website/docs/r/logging_project_exclusion.html.markdown

```diff
@@ -0,0 +1,55 @@
+---
+layout: "google"
+page_title: "Google: google_logging_project_exclusion"
+sidebar_current: "docs-google-logging-project-exclusion"
+description: |-
+  Manages a project-level logging exclusion.
+---
+
+# google\_logging\_project\_exclusion
+
+Manages a project-level logging exclusion. For more information see
+[the official documentation](https://cloud.google.com/logging/docs/) and
+[Exclusing Logs](https://cloud.google.com/logging/docs/exclusions).
+
+Note that you must have the "Logs Configuration Writer" IAM role (`roles/logging.configWriter`)
+granted to the credentials used with terraform.
+
+## Example Usage
+
+```hcl
+resource "google_logging_project_exclusion" "my-exclusion" {
+    name = "my-pubsub-instance-exclusion"
```

Comment:
I know it's just an example config but it does seem a bit out of place to have the name be about pubsub and the description/filter be about GCE